test/jest/__mock__/stripesComponents.mock.js

```diff
@@ -33,9 +48,23 @@ jest.mock('@folio/stripes/components', () => ({
       {children}
     </div>
   )),
-  ConfirmationModal: jest.fn(() => <div>ConfirmationModal</div>),
+  ConfirmationModal: jest.fn(({ heading, confirmLabel, cancelLabel, onConfirm, onCancel }) => (
+    <div>
+      <span>ConfirmationModal</span>
+      <div>
+        <div>{heading}</div>
+        <button type="button" onClick={onConfirm}>{confirmLabel}</button>
+        <button type="button" onClick={onCancel}>{cancelLabel}</button>
+      </div>
+    </div>)),
   Datepicker: jest.fn(() => <div>Datepicker</div>),
-  ErrorModal: jest.fn(() => <div>ErrorModal</div>),
+  ErrorModal: jest.fn(({ label, content, buttonLabel, onClose }) => (
```

Comment:
I do not see the usage of this component inside `PickupServicePointFilter`. Do we really need to mock it?